tests/test_algorithm.py

```diff
@@ -4618,47 +4647,48 @@ def test_minutely_delisted_equities(self):
 
         self.assertEqual(algo.num_positions, expected_position_counts)
         self.assertEqual(
-            list(output['longs_count']),
+            output.daily_performance.longs_count.tolist(),
             [3, 3, 3, 2, 2, 1, 1],
         )
 
         # Check expected transactions.
         # We should have a transaction of order_size shares per sid.
-        transactions = output['transactions']
+        transactions = output.transactions
 
         # Note that the transactions appear on the first day rather than the
         # second in minute mode, because the fills happen on the second tick of
         # the backtest, which is still on the first day in minute mode.
-        initial_fills = transactions.iloc[0]
+        initial_fills = transactions.loc[resources.sim_params.start_session]
         self.assertEqual(len(initial_fills), len(assets))
-        for asset, txn in zip(assets, initial_fills):
-            self.assertDictContainsSubset(
-                {
+        assert_equal(
+            initial_fills.drop('order_id', axis=1),
+            pd.DataFrame(
+                index=[start_session] * 3,
+                data={
                     'amount': order_size,
-                    'commission': None,
-                    'dt': backtest_minutes[1],
-                    'price': initial_fill_prices[asset],
-                    'sid': asset,
-                },
-                txn,
+                    'commission': np.nan,
```

Comment:
Where does this change from `None` to `nan`?

Reply:
The old test for this was reading the dictionaries emitted by the algorithm generator, which contained None.  The new test is reading from the dataframe produced by AlgorithmResult, which feeds all the extracted values into `np.array(..., dtype=float)`.